### ISSUE TITLE
Can't open file from google drive

### ISSUE THREAD

**Issue by Blaaznar** [MiniKeePass/MiniKeePass#459]
MiniKeePass app doesn't show up in the 'Open in' list.

**Comment by mattdreher** [comment]
I have had the same problem.

**Comment by novityFS** [comment]
This has worked some time ago. It broke with an update, but I don't know if Drive or MiniKeePass is responsible for the problem.

**Comment by utilman** [comment]
No.. it is not a MiniKeePass problem. Some cloud providers like Google are stripping the extension.
So if you rename your database to name..kdbx.kdbx and open it from the Google drive you can choose the 'Open in' again

**Comment by brainchain** [comment]
It's not just google drive. I try to open from Dropbox too. I tried the file renaming but it still doesn't work. This is bad, i used to recommend this app to all my customers. iPhone. 5s 9.3.5

**Comment by gwerbin** [comment]
Just stopped working with the MEGA app after updating to 10.2.
